src/WebApp/flask/templates/layout.html

```diff
@@ -98,13 +103,17 @@
     }).change(function(e) {
       //ws.send(e.value.newValue);
     });
-*/    
-    $.get('/.auth/me').done(function (data) {
-      var user_name = data[0].user_claims.find(uc => uc.typ === 'name').val;
-      $('#user_menu').text(user_name);
-    });
+*/ 
+
+      if( '{{aztkClusterStatus}}' == 'Provisioning' || '{{aztkClusterStatus}}' == 'Deleting')
```

Comment:
this code should not be in the layout. why would you want to run it for ALL pages at ALL times?